### ISSUE TITLE
Redmine\Api\SimpleXMLElement not working properly with PHP7

### ISSUE THREAD

**Issue by mark9000** [kbsali/php-redmine-api#162]
The addChild method in the Redmine\Api\SimpleXMLElement class does not properly in PHP7. The content of "notes" is not longer a string.

Code example:
`$xml = new Redmine\Api\SimpleXMLElement('<?xml version="1.0"?><issue></issue>');
$xml->addChild('notes', 'Hello, world!');
echo $xml->asXml();`
Output with PHP5.6:
`<?xml version="1.0"?>
<issue><notes>Hello, world!</notes></issue>`
Output with PHP7:
`<?xml version="1.0"?>
<issue><notes><0>Hello, world!</0></notes></issue>`

Thanks!

**Comment by xl32** [comment]
It works for me if I change  Redmine\Api\SimpleXMLElement to stub class:

```
<?php

namespace Redmine\Api;

class SimpleXMLElement extends \SimpleXMLElement
{
}
```

**Comment by mark9000** [comment]
That is the same solution i found!

**Comment by xl32** [comment]
Added [pull request](https://github.com/kbsali/php-redmine-api/pull/164)

**Comment by xl32** [comment]
Pull request #164 is merged, so I think this is fixed now.

**Comment by kbsali** [comment]
@mark9000 closing as fixed in #164 (please re-open if this is not the case)

Thanks!
